packages/vue-server-renderer/types/index.d.ts

```diff
@@ -0,0 +1,44 @@
+import Vue = require('vue');
+import { Readable } from 'stream';
+
+export declare function createRenderer(options?: RendererOptions): Renderer;
+
+export declare function createBundleRenderer(bundle: any, options?: BundleRendererOptions): BundleRenderer;
```

Comment:
http://ssr.vuejs.org/en/api.html#createbundlerendererbundle-options

IMHO a `bundle` is either a `string` or an `object`.